thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/db/entity/WebappConfig.java

```diff
@@ -0,0 +1,95 @@
+package com.linkedin.thirdeye.db.entity;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import com.google.common.base.MoreObjects;
+import com.linkedin.thirdeye.dashboard.configs.WebappConfigClassFactory.WebappConfigType;
+
+/**
+ * Entity class for webapp configs. config_id, collection, config_type conbination should be unique
+ */
+@Entity
+@Table(name = "webapp_configs",
+  uniqueConstraints = {@UniqueConstraint(columnNames={"config_id", "collection", "config_type"})})
+public class WebappConfig  extends AbstractBaseEntity {
+
+  @Column(name = "config_id", nullable = false)
```

Comment:
dashboard configs can have same collection and config type, but different ids. This config_id is necessary to ensure that only 1 collectionConfig and CollectionSchema is allowed per collection, but multiple are allowed for dashboardConfig. There could be more configs in future with same requirement. Please read design doc sent with PR for explanation